src/palette/qml/MuseScore/Palette/PaletteTree.qml

```diff
@@ -29,7 +29,7 @@ import "utils.js" as Utils
 
 ListView {
     id: paletteTree
-    Accessible.name: qsTrc("palette", "Palettes Tree, contains %n palette(s)").arg(count)
+    Accessible.name: qsTrc("palette", "Palettes Tree, contains %1 palette(s)").arg(count)
```

Comment:
Ah, that is what it means...!
I thought it was a mistake because there was a warning and I could not find any documentation about `%n`. Now I have finally found [some documentation](https://doc.qt.io/qt-5/i18n-source-translation.html#handling-plurals), so I will have a closer look at it.

Reply:
I found out what was the problem! Thanks Jojo, for making me aware of this!
In 3.x, this problem does not occur, so no need to change anything there.